client_tools.go

```diff
@@ -140,21 +171,3 @@ func (ops *Operations) Delete(field int, arg interface{}) *Operations {
 func (ops *Operations) Assign(field int, arg interface{}) *Operations {
 	return ops.append(assignOperator, field, arg)
 }
-
-type OpSplice struct {
```

Comment:
Please add a note to README.md into the migration article about the change in the public API.